core/layout_api.php

```diff
@@ -538,13 +538,22 @@ function layout_navbar_button_bar() {
 		return;
 	}
 
+	$t_can_report_bug = access_has_project_level( config_get( 'report_bug_threshold' ) );
+	$t_can_invite_user = current_user_is_administrator();
```

Comment:
use $t_can_invite_user also at line 558 to replace call of current_user_is_administrator()